modUtil/src/org/aion/util/bytes/ByteUtil.java

```diff
@@ -624,19 +616,19 @@ public static void intsToBytes(int[] arr, byte[] b, boolean bigEndian) {
             int off = 0;
             for (int i = 0; i < arr.length; i++) {
```

Comment:
additional update: could be changed to `for (int ii : arr)`; same for the loop below